packages/backend-common/src/database/DatabaseManager.ts

```diff
@@ -187,17 +204,30 @@ export class DatabaseManager {
     };
   }
 
+  /**
+   * Provides a partial Knex.Config database schema override for a given plugin.
+   *
+   * @param pluginId Target plugin to get database schema override
+   * @returns Partial Knex.Config with database schema override
+   */
+  private getSchemaOverrides(pluginId: string): Knex.Config | undefined {
+    return createSchemaOverride(this.getClientType(pluginId).client, pluginId);
+  }
+
   /**
    * Provides a partial Knex.Config database name override for a given plugin.
    *
    * @param pluginId Target plugin to get database name override
    * @returns Partial Knex.Config with database name override
    */
   private getDatabaseOverrides(pluginId: string): Knex.Config {
-    return createNameOverride(
-      this.getClientType(pluginId).client,
-      this.getDatabaseName(pluginId),
-    );
+    const databaseNameOverride = this.getDatabaseName(pluginId);
```

Comment:
Perhaps call the variable `databaseName` here to avoid confusion

Reply:
Fixed, thanks!! 😁